### PR TITLE
ID-58

### PR DESCRIPTION
**Pull request by xiangzhe0812**
eject ios and android, allows react-native-fast-pgp package working good on native ios app

### CODE REVIEW

File: app/android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<resources>
+  <string name="app_name">blockchain-id</string>
```
Comment (bitcoinbrisbane) [inline app/android/app/src/main/res/values/strings.xml]:
```suggestion
  <string name="app_name">idem</string>
```

File: app/screens/setting/PgpSetting.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        }}
+      />
+      <TextInput
+        style={{
+          ...styles.claim.input,
+          width: styles.layout.window.width,
+          marginTop: 10,
+        }}
+        keyboardType="default"
+        placeholder="Please enter your mnemonic key"
```
Comment (bitcoinbrisbane) [inline app/screens/setting/PgpSetting.tsx]:
```suggestion
        placeholder="Please enter your mnemonic seed phrase"
```

**Review by bitcoinbrisbane — COMMENTED** [review]
Few small things, other wise well done.

**Review by ahinchliff — APPROVED** [review]
with expo eject come great responsibility!

**Review by rellfy — APPROVED** [review]

**Pull request merged**